### PR TITLE
en: update view logs and collect logs

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Update view logs and collect logs.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/339
- Other reference link(s):

### CODE REVIEW

File: en/view-logs.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: View TiDB Logs in Kubernetes
+summary: Learn how to view TiDB slow logs and application logs in Kubernetes.
+category: how-to
+aliases: ['docs/tidb-in-kubernetes/dev/collect-tidb-logs/']
```
Comment (yikeke) [inline en/view-logs.md:5]:
```suggestion
aliases: ['/docs/tidb-in-kubernetes/dev/collect-tidb-logs/']
```
The alias must start with a slash?

File: zh/view-logs.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 title: 查看日志
 summary: 介绍如何查看 TiDB 集群各组件日志以及 TiDB 慢查询日志。
 category: how-to
+aliases: ['docs-cn/tidb-in-kubernetes/dev/collect-tidb-logs/']
```
Comment (yikeke) [inline zh/view-logs.md:5]:
```suggestion
aliases: ['/docs-cn/tidb-in-kubernetes/dev/collect-tidb-logs/']
```

File: en/TOC.md [hunk 3]
```diff
@@ -71,3 +71,4 @@
     - [Configure tidb-drainer Chart](configure-tidb-binlog-drainer.md)
     - [Configure tidb-cluster Chart](tidb-cluster-chart-config.md)
     - [Configure tidb-backup Chart](configure-backup.md)
+  - [Log Collection](logs-collection.md)
```
[the remark this answers is not in the document]
Comment (ran-huang) [reply en/TOC.md:74]: I agree. I'll update the Chinese file name and add an alias.

File: en/view-logs.md (hunk 1 above)
Comment (ran-huang) [reply en/view-logs.md:5]: You're right. I'll update them later.

**Comment by ran-huang** [comment]
@weekface PTAL. Thanks!

**Pull request merged**